source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -141,5 +141,10 @@ AndroidContext: class extends OpenGLContext {
 		map add("startY", startY)
 		DrawState new(target) setMap(map) draw()
 	}
+	preallocate: override func (resolution: IntVector2D) { GraphicBufferYuv420Semiplanar free~all() }
+	preregister: override func (image: Image) {
+		if (image instanceOf(GraphicBufferYuv420Semiplanar))
+			(image as GraphicBufferYuv420Semiplanar) toRgba(this) referenceCount decrease()
```

Comment:
You might have to re-test this later since I am not sure if YUV to RGBA works right at this moment.